Hibernate/hibernateCRUD/UpdateStudentDemo.java

```diff
@@ -0,0 +1,37 @@
+package hibernateCRUD;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+import hibernateAnnotationDemo.Student;
+
+public class UpdateStudentDemo {
+
+	public static void main(String[] args) {
+		SessionFactory factory = new Configuration()
+				.configure("hibernate.cfg.xml")
+				.addAnnotatedClass(Student.class)
+				.buildSessionFactory();
+
+		Session session = factory.getCurrentSession();
+		try {
+			int studentId = 1;
+			session = factory.getCurrentSession();
+			session.beginTransaction();
+			System.out.println("\nGetting student with id: " + studentId);
+			Student myStudent = session.get(Student.class, studentId);
+//			System.out.println("Deleting student: " + myStudent);
+//			session.delete(myStudent);
```

Comment:
It's always good practice to use session built in methods for delete rather then executeUpdate()

Reply:
done